README_GPDB_en.md

```diff
@@ -120,7 +165,7 @@ gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" /usr/l
 gpstop -ar
 ```
 
-- Install the gp_pitr extension
+- Install the gp_pitr extension (skip this step for GreenPlum 7)
```

Comment:
At the all places you wrote "Greenplum", but here you wrote "GreenPlum" (and at the same place in README_GPDB_ru.md).

Reply:
Thanks! I've fixed it.